### PR TITLE
docs: add login help article for SBID

### PR DESCRIPTION
**Pull request by mefellows**

### CODE REVIEW

**Comment by mefellows** [comment]
@AnubhutiShrivastava noticed an additional page that needed updating.

File: website/docs/docs/login.md [hunk 1]
```diff
@@ -5,9 +5,19 @@ title: Login
 
 This section describes certain issues you might face while logging into PactFlow and explains how to resolve them.
 
-## Unable to login into PactFlow user interface
+## *"You are not authorized to access this Pactflow account"*
 
-When you sign up for PactFlow, you will be emailed the login details with a temporary password. This password is valid for 7 days. If you need to reset the password, you can reset the clock on this expiry by simply re-inviting the user. Alternatively, [contact us (https://support.smartbear.com/pactflow/message/) and we will reset it for you.
```
Comment (AnubhutiShrivastava) [inline website/docs/docs/login.md:10]: When you sign up for PactFlow, you will get an email comprising the login details with a temporary password.

File: website/docs/docs/login.md [hunk 2]
```diff
@@ -5,9 +5,19 @@ title: Login
...
+
+### 1. Your email has not been confirmed
+
+Please check your inbox for an email with the subject "Verify your email" from `no-reply@id.smartbear.com`, and choose "confirm my account".
+
+After this, you should be able to login.
+
+### 2. You have not been invited to the account
+
+Only users that created the account (initial user) or have been invited to an account are able to login.
```
Comment (AnubhutiShrivastava) [inline website/docs/docs/login.md:20]: Users who created the account (initial users) or received an invitation to an account are able to login.

**Review by AnubhutiShrivastava — COMMENTED** [review]
Finished review: rephrased the content for active voice and grammar precision.

**Pull request merged**